src/services/commentServices.js

```diff
@@ -0,0 +1,17 @@
+import models from '../models';
+
+/**
+ * Comment service Class
+ */
+export default class commentServices {
+  /**
+   * @name CreateComment
+   * @description Interacts with model to create comment
+   * @param { string } userData the user's data
```

Comment:
I think it's better you use comment here instead of userData

Reply:
I used that because I'm also sending in some other details too, userId inclusive

Reply:
Nice setup